src/components/admin/ChangesViewer/DiffView.tsx

```
@@ -0,0 +1,333 @@
+
+    // Check if this is a translatable field (value is a translation object)
+    const isTranslatable = isTranslationObject(newValue);
+
+    if (field.type === 'repeater') {
+        // ... repeater logic using oldValue/newValue ...
+        // Simplicity: Show simplified diff for Repeater
+        const isModified = JSON.stringify(oldValue) !== JSON.stringify(newValue);
+        return (
+            <div className="grid grid-cols-2 gap-8 py-4 border-b last:border-0 items-start">
+                <div className="opacity-70 pointer-events-none">
+                    <label className="text-sm font-medium mb-2 block">{fieldName} (Old)</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)